### PR TITLE
adding an imprint and our registration number to the about page

### PR DESCRIPTION
**Pull request by merchantcoppola**

### CODE REVIEW

**Review by melMass — APPROVED** [review]
👍 I did not follow all, but AFAIK imprint's must also have the representative's name

File: src/lang/en/about.md [hunk 1]
```diff
@@ -14,42 +14,54 @@
...
-**Marketplace Contract:** [KT1PHubm9HtyQEJ4BBpMTVomq6mhbfNZ9z5w](https://tzkt.io/KT1PHubm9HtyQEJ4BBpMTVomq6mhbfNZ9z5w/operations/) by [Teia community](https://github.com/teia-community/teia-smart-contracts/blob/main/python/contracts/teiaMarketplace_v1.py) - audited by [Inference AG](https://github.com/InferenceAG/ReportPublications/blob/master/Inference%20AG%20-%20Teia%20community%20-%20marketplace%20%26%20multisig%20-%20v1.0.pdf)
+## Tech Specs
 
-**Marketplace Contract Admin:** [KT1PKBTVmdxfgkFvSeNUQacYiEFsPBw16B4P](https://tzkt.io/KT1PKBTVmdxfgkFvSeNUQacYiEFsPBw16B4P/operations/) - [multisig contract](https://multisign.onrender.com/) developed and owned by the Teia Community
+The Teia Marketplace Interface and marketplace contract are code forks based on the open source Tezos NFT marketplace hic et nunc that have been and will be further developed and maintained by the Teia community. 
+
+The Teia Marketplace Interface and marketplace contract are [code forks](https://github.com/teia-community/teia-ui) based on the open source Tezos NFT marketplace [hic et nunc](https://github.com/hicetnunc2000) that have been and will be further developed and maintained by the Teia community.
```
Comment (Zir0h) [inline src/lang/en/about.md]: this is 2 times the same sentence?

**Comment by merchantcoppola** [comment]
> 👍 I did not follow all, but AFAIK imprint's must also have the representative's name

depends on the jurisdiction afaik, from what i researched, RMI doesnt require us to have a imprint legally, i just wanted to add the reg number there because some people started to ask for it for their tax filings, we can add representatives later on if needed ...

i think for GDPR compliance we would need to add a rep name or address  indeed, but lets postponme this for later

File: src/lang/en/about.md (hunk 1 above)
Comment (merchantcoppola) [reply src/lang/en/about.md]: yep, removed the fisrt one, thanks"

**Pull request merged**